modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -103,8 +103,21 @@ label:new[]
 DROP [COMPOSITE] DATABASE ... [RESTRICT \| CASCADE ALIAS[ES]]
 ----
 
-| Added the ability to drop any database aliases, that would otherwise block the deletion of the database, at the same time as the deletion of the database.
+| Added the ability to drop database aliases while deleting a database.
```

Comment:
```suggestion
| Added the ability to drop database aliases while dropping a database.
```
maybe?

Reply:
though I guess we are already pretty inconsistent if we write drop or delete 🤷

Reply:
You are right - we need to decide on which to use and stick with it. Personally I think we should use drop because that is what the command is called (otherwise I definitely would have preferred "delete"). I will bring this up with my Ops Manual colleagues and take a decision.